### PR TITLE
feat(dapp): story feed

### PR DESCRIPTION
**Pull request by sruddy**

### CODE REVIEW

File: packages/components/src/StoryFeed/Story.tsx [hunk 1]
```diff
@@ -13,11 +12,11 @@ export const Story: React.FunctionComponent<StoryProps> = props => {
     <>
       <StoryLink onClick={props.handleOpenStory}>
         <StoryLinkLeft>
-          <StoryTitle>{props.title}</StoryTitle>
-          <TimeStamp>{props.timeStamp}</TimeStamp>
+          <StoryTitle>{props.openGraphData.title}</StoryTitle>
+          <TimeStamp>{props.openGraphData.article.published_time}</TimeStamp>
```
Comment (nickreynolds) [inline packages/components/src/StoryFeed/Story.tsx]: I don't think we should use the open graph published time here, we should use the post created-at time

File: packages/components/src/StoryFeed/StoryFeed.stories.tsx [hunk 2]
```diff
@@ -16,6 +16,31 @@ const contributors = [
...
+  },
+};
+
+const openGraphData = {
+  article: {
+    published_time: "2019-08-28T15:58:32Z",
+  },
+  description: "Uyghurs are gaming TikTok’s algorithm to find a loophole in Xinjiangs’s information lockdown",
+  images: {
+    url: "https://codastory.com/wp-content/uploads/2019/09/Still-Header-.png",
```
Comment (nickreynolds) [inline packages/components/src/StoryFeed/StoryFeed.stories.tsx]: if this were to match how it's stored on the backend, images is actually an array of objects. so accessing the image would be images[0].url

File: packages/components/src/StoryFeed/types.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    };
+  };
+}
+
+export interface OpenGraphData {
+  article: {
+    published_time: string;
+  };
+  description: string;
+  images: {
```
Comment (nickreynolds) [inline packages/components/src/StoryFeed/types.ts]: left this comment elsewhere, but makes more sense here. if this were to match how it's stored on the backend, images is actually an array of objects. so accessing the image would be images[0].url

File: packages/components/src/StoryFeed/Story.tsx (hunk 1 above)
Comment (sruddy) [reply packages/components/src/StoryFeed/Story.tsx]: @nickreynolds Is post created at the time of link submission or the time the article was published?

Comment (sruddy) [reply packages/components/src/StoryFeed/Story.tsx]: there's a conversation in slack pod-pulse about which we should use

**Comment by dankins** [comment]
:tada: This PR is included in version 4.8.23 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

**Pull request merged**